### DISCUSSION TITLE
PNG to WEBP

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7177] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

How do I losslessly convert png files to non-animated webp files?

**Comment by fmw42** [comment]
`magick wizard.png -quality 100 -define webp:lossless=true wizard.webp`

See https://imagemagick.org/script/webp.php

**Reply by Jay123210599** [reply]
It only converted one png file.

**Reply by fmw42** [reply]
You did not say multiple inputs.  For that you need to use mogrify.

`mogrify -format webp -quality 100 -define webp:lossless=true *.png`
